src/domains/ecommerce-app/api/ecommerce-checkout/v1/_openapi.json.tpl

```diff
@@ -1793,13 +1793,18 @@
             "items": {
               "$ref": "#/components/schemas/TransferListItem"
             }
+          },
+          "allCCP": {
+            "description": "Flag for the inclusion of Poste bundles. false -> excluded, true -> included",
+            "type": "boolean"
           }
         },
         "required": [
           "paymentAmount",
           "primaryCreditorInstitution",
           "transferList",
-          "touchpoint"
+          "touchpoint",
+          "allCCP"
```

Comment:
not required :-) but _isAllCCP_ as param name could refer to the boolean type

Reply:
Did it!